documentation/modules/rn-2.6.adoc

```diff
@@ -123,72 +136,29 @@ With this update, you can edit plans that have failed to migrate any VMs. Some p
-
-.Migration fails during precopy/cutover while performing a snapshot operation on the source VM
-
-Warm migration from {rhv-short} fails if a snapshot operation is triggered and running on the source VM at the same time as the migration is scheduled. The migration does not wait for the snapshot operation to finish. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
-
-.Unable to schedule migrated VM with multiple disks to more than one storage class of type `hostPath`
-
-When migrating a VM with multiple disks to more than one storage class of type `hostPath`, it might happen that a VM cannot be scheduled. Workaround: Use shared storage on the target {ocp} cluster.
-
-.Non-supported guest operating systems in warm migrations
-
-Warm migrations and migrations to remote {ocp} clusters from vSphere do not support the same guest operating systems that are supported in cold migrations and migrations to the local {ocp} cluster. RHEL 8 and RHEL 9 might cause this limitation.
-
-See link:https://access.redhat.com/articles/1351473[Converting virtual machines from other hypervisors to KVM with virt-v2v in RHEL 7, RHEL 8, and RHEL 9] for the list of supported guest operating systems.
-
-.VMs from vSphere with RHEL 9 guest operating system can start with network interfaces that are down
-
-When migrating VMs that are installed with RHEL 9 as a guest operating system from vSphere, the network interfaces of the VMs could be disabled when they start in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-491[(MTV-491)]
-
-
-.Migration of a VM with NVME disks from vSphere fails
-
-When migrating a virtual machine (VM) with NVME disks from vSphere, the migration process fails, and the Web Console shows that the `Convert image to kubevirt` stage is `running` but did not finish successfully. link:https://issues.redhat.com/browse/MTV-963[(MTV-963)]
-
-.Importing image-based VMs can fail
-
-Migrating an image-based VM without the `virtual_size` field can fail on a block mode storage class. link:https://issues.redhat.com/browse/MTV-946[(MTV-946)]
-
-.Deleting a migration plan does not remove temporary resources
-
-Deleting a migration plan does not remove temporary resources such as importer pods, conversion pods, config maps, secrets, failed VMs, and data volumes. You must archive a migration plan before deleting it to clean up the temporary resources. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
-
-.Migrating VMs with independent persistent disks from VMware to OCP-V fails
+[id="resolved-issues-26_{context}"]
+== Resolved issues
 
-Migrating VMs with independent persistent disks from VMware to OCP-V fails. link:https://issues.redhat.com/browse/MTV-993[(MTV-993)]
+This release has the following resolved issues:
 
-.Guest operating system from vSphere might be missing
+[id="resolved-issues-26-4_{context}"]
+=== Resolved issues 2.6.4
```

Comment:
We are also fixing

https://issues.redhat.com/browse/MTV-1299

Please add a section for this.

Reply:
MTV-1299 added to Resolved Issues 

| |
| --- |
| ![image](https://github.com/user-attachments/assets/36f51df9-703f-43f3-8492-fed41faa0b88) |

Are we ok to keep the workaround?

Reply:
No, please remove it

Reply:
… because we did not focus on it in testing.

Reply:
Removed